common/wforce-webserver.cc

```diff
@@ -379,9 +381,10 @@ void WforceWebserver::pollThread(WforceWebserver* wws)
               wws->d_queue.push(wqi);
             }
           }
-          wws->d_cv.notify_all();
 	}
       }
+      if (pollin)
+        wws->d_cv.notify_all();
```

Comment:
why not just `if (wws->d_queue.size() > 0)`

Reply:
Hmm that's a good point. Will make that change I think